contributors/design-proposals/kubelet-authorizer.md

```diff
@@ -146,29 +142,39 @@ for compatibility with deployment methods that bind other users or groups to tha
 
 ### Kubelets outside the `system:nodes` group
 
-Kubelets outside the `system:nodes` group would not be authorized by the node authorizer,
+Kubelets outside the `system:nodes` group would not be authorized by the `Node` authorization mode,
 and would need to continue to be authorized via whatever mechanism currently authorizes them.
 The node admission plugin would not restrict requests from these kubelets.
 
 ### Kubelets with undifferentiated usernames
 
 In some deployments, kubelets have credentials that place them in the `system:nodes` group,
 but do not identify the particular node they are associated with.
-Those kubelets would be broadly authorized by the node authorizer,
-but would not have secret and configmap requests restricted.
-The node admission plugin would not restrict requests from these kubelets.
+
+These kubelets would not be authorized by the `Node` authorization mode,
+and would need to continue to be authorized via whatever mechanism currently authorizes them.
+
+The node admission plugin *would* restrict requests from these kubelets.
+That plugin should not be used in a deployment with kubelets that do not identify themselves.
```

Comment:
What are the implications of running the node authorizer without the admission piece? I think the lack of the admission plugin enables a simple way to bypass the authorizer (e.g. create a fake pod bound to itself that references the secret you want to read), right?

Given that there isn't a good fallback option for admission, would it make sense to keep the non-strict mode option there?